Cargo.lock

```diff
@@ -1714,9 +1714,9 @@ dependencies = [
 
 [[package]]
 name = "itertools"
-version = "0.10.0"
+version = "0.10.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "37d572918e350e82412fe766d24b15e6682fb2ed2bbe018280caa810397cb319"
+checksum = "69ddb889f9d0d08a67338271fa9b62996bc788c7796a5c18cf057420aaed5eaf"
```

Comment:
nit: I don't think you meant to update itertools in this PR?

Reply:
That's right... How does one un-update a crate?

Reply:
That's right...